pkg/utils/functional/suite_test.go

```diff
@@ -0,0 +1,74 @@
+package functional
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestFunctional(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Functional Suite")
+}
+
+var _ = Describe("Functional", func() {
+	Context("UnionStringMaps", func() {
+		empty := make(map[string]string)
```

Comment:
I'd probably just define the maps inline. [string]string{} is only a few extra characters and more explicit.